drain-service/drain3/drain.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, log_template_tokens: list, cluster_id: int, is_pretrained = F
         self.pretrained_anomaly_level = pretrained_anomaly_level
         self.cache = dict()
         self.size = 1
+        self.max_frequency_original_log = ""
```

Comment:
it's not needed. bc 

 self.max_frequency_original_log is equal to self.cache[self.max_frequency_masked_log][message]